druid-shell/src/platform/windows/application.rs

```diff
@@ -105,6 +105,11 @@ impl Application {
         unsafe {
             // Handle windows messages
             loop {
+                if let Ok(state) = self.state.try_borrow() {
+                    for hwnd in &state.windows {
+                        SendMessageW(*hwnd, DS_HANDLE_DROPPED, 0,0);
+                    }
+                }
```

Comment:
Sends a custom message each loop to handle dropped WM_ messages

Reply:
I'm not convinced that it's worth putting in a temporary solution for this, given that we want a more permanent one soonish anyway.

Actually, this looks like a good place (and mechanism) for handling the permanent solution: the only difference is that we queue up blocking operations instead of messages: so there'd be an
```
enum BlockingOp {
   SaveDialog(...),
   LoadDialog(...),
   SetSize(...),
   // anything else?
}
```

Each window would queue up these operations and run them on `DS_HANDLE_DROPPED`

Reply:
How would you detect when the handle is dropped?

Reply:
At this point in the event loop, it should always be dropped. That is, we'll ensure that we never return to this loop while holding a `RefCell` borrow.